mlir/include/mlir/Dialect/MIOpen/XdlopsCodeSelection.h

```diff
@@ -658,7 +664,7 @@ struct XdlopsCodeSelection {
       cycles = 8;
       k_base = 2;
     } else {
-      llvm::errs() << "Unsupported case as mfmaInstr not selected!\n";
+      llvm_unreachable("Unsupported case as mfmaInstr not selected!\n");
```

Comment:
Will this crash the program, and would you suggest using this to intentionally crash ourselves? I figure that our project overall doesn't yet have a consistent way to handle errors.